### PR TITLE
fix(a11y): making image captions their alt text

### PR DESCRIPTION
**Pull request by melodysifry**
[][demo] | Fixes RM-1562
:---:|:---:

## 🧰 Changes

- if a caption is provided for an image, but no alt text, use the caption text to populate the alt text for an image
- we'll want to revisit this at some point and add proper alt text support to avoid potentially [repeating alt text and caption text](https://www.brandeis.edu/web-accessibility/understanding/images-alt-text.html)

## 🧬 QA & Testing

- Does alt text appear now for images that have captions?

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-231.herokuapp.com/#images
[prod]: https://developers.readme.io/docs

### CODE REVIEW

**Comment by billhimmelsbach** [comment]
Updated the description some for ya! 👍

**Review by rafegoldberg — APPROVED** [review]

**Review by billhimmelsbach — APPROVED** [review]
Looks great! 🎉

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.30.2-next.1</b>](/readmeio/markdown/releases/tag/v6.30.2-next.1)
<br>

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.30.2</b>](/readmeio/markdown/releases/tag/v6.30.2)
<br>

File: __tests__/__snapshots__/flavored-compilers.test.js.snap [hunk 1]
```diff
@@ -36,7 +36,7 @@ exports[`ReadMe Magic Blocks Figure 1`] = `
 
 <figure>
 
-![](https://files.readme.io/6f52e22-man-eating-pizza-and-making-an-ok-gesture.jpg \\"rdme-blue.svg\\")
+![Ok, pizza man.](https://files.readme.io/6f52e22-man-eating-pizza-and-making-an-ok-gesture.jpg \\"rdme-blue.svg\\")
```
Comment (erunion) [inline __tests__/__snapshots__/flavored-compilers.test.js.snap:39]: 🤣

**Pull request merged**